steps/create-hive-tables.py

```diff
@@ -1,11 +1,12 @@
 import boto3
 import ast
+import os
 from logger import setup_logging
 
-level = "info"
-logger_path = "/var/log/adg/hive_tables_creation_log.log"
-logger_format = "{ 'timestamp': '%(asctime)s', 'log_level': '%(levelname)s', 'message': '%(message)s' }"
-the_logger = setup_logging(level, logger_path , logger_format)
+the_logger = setup_logging(
+    log_level=os.getenv("ADG_LOG_LEVEL"),
```

Comment:
this will fail if env var is missing or not uppercase, so I suggest the following:
```suggestion
    log_level = os.environ["ADG_LOG_LEVEL"].upper() if "ADG_LOG_LEVEL" in os.environ else "INFO"
```